src/input.rs

```diff
@@ -161,6 +161,27 @@ impl Opts {
         opts
     }
 
+    pub fn default() -> Self {
```

Comment:
This is the only major change (as other tests reference this, they no longer work).

Reply:
I could move this to a `impl Default for Opts` which resolves the `clippy` warning, unless there's a better way…?

Reply:
Added `impl Default for Opts`.